spec/integration/workflows-steps-inputs.feature

```diff
@@ -12,3 +12,67 @@ Feature: Workflow steps inputs
     And "Step Versions" should "be visible"
     And "Step Inputs" should "be visible"
     And "Save Button" should "be disabled"
+
+  Scenario: Inputs without categories don't get wrapped into expandable container
+    When I click on "First step"
+    Then "Step Inputs Without Category" should "be visible"
+
+  Scenario: Inputs with categories get wrapped into expandable container
+    When I click on "Second step"
+    Then I should see "Config" in "Second Step Input Category Title"
```

Comment:
can't we use generic element here like `Selected Step Input Category Title`? I believe only one input element will present in the DOM at a single time and that's only the selected, or am I mistaken? Or is it purely a naming issue?

Reply:
These are all about the second input in the selected step (not the second step). I did think about naming it `Selected Step Second Input ...` WDYT @tadam313 ?

Reply:
I'd suggest to do that yeah, I think it would be more clear what it's referring to